samples/java_layout/src/main/java/io/customer/android/sample/java_layout/data/model/CustomerIOSDKConfig.java

```diff
@@ -0,0 +1,160 @@
+package io.customer.android.sample.java_layout.data.model;
+
+import android.text.TextUtils;
+
+import androidx.annotation.NonNull;
+import androidx.annotation.Nullable;
+
+import java.util.HashMap;
+import java.util.Map;
+
+import io.customer.android.sample.java_layout.BuildConfig;
+import io.customer.android.sample.java_layout.core.StringUtils;
+import io.customer.android.sample.java_layout.support.Optional;
+
+public class CustomerIOSDKConfig {
+    private static class Keys {
+        static final String SITE_ID = "cio_sdk_site_id";
+        static final String API_KEY = "cio_sdk_api_key";
+        static final String TRACKING_URL = "cio_sdk_tracking_url";
+        static final String BQ_SECONDS_DELAY = "cio_sdk_bq_seconds_delay";
+        static final String BQ_MIN_TASKS = "cio_sdk_bq_min_tasks";
+        static final String ENABLE_IN_APP = "cio_sdk_enable_in_app";
+        static final String TRACK_SCREENS = "cio_sdk_track_screens";
+        static final String TRACK_DEVICE_ATTRIBUTES = "cio_sdk_track_device_attributes";
+        static final String DEBUG_MODE = "cio_sdk_debug_mode";
+    }
+
+
+    public static CustomerIOSDKConfig getDefaultConfigurations() {
+        return new CustomerIOSDKConfig(BuildConfig.SITE_ID,
+                BuildConfig.API_KEY,
+                BuildConfig.TRACKING_URL,
+                BuildConfig.BQ_SECONDS_DELAY,
+                BuildConfig.BQ_MIN_TASKS,
+                BuildConfig.ENABLE_IN_APP,
+                BuildConfig.TRACK_SCREENS,
+                BuildConfig.TRACK_DEVICE_ATTRIBUTES,
+                BuildConfig.DEBUG_MODE);
+    }
```

Comment:
I would prefer if the default configuration is not the configuration at compile time of this app but the default configuration used by the CIO SDK.

Reply:
I would agree. Having them in `BuildConfig` seems an overkill when we have defaults and settings screen to update them?

Reply:
They are added mainly to [facilitate sample apps testing](https://github.com/customerio/customerio-android/pull/198#discussion_r1188339976). If we remove that and still want to reduce test time, we'll have to open settings screen and update configurations before every test which isn't definitely ideal and I don't like. But if you both think we should remove default values, I can remove that.

Please note that removing default values won't change anything in SDK initialization and all values are completely optional except `siteId` and `apiKey`.

Reply:
It's a useful feature but I think QA mostly tests apps from Firebase anyways, they don't usually test it locally. Even if they want to update them locally then can probably change file with static fallbacks?

Reply:
Ah, yes. I was not understanding that this feature was built to improve the appium experience. I was thinking it was for QA.

Reply:
Yeah, I understand now. But since I've already reverted this, we can make desired changes when integrating appium tests with sample apps.

Reply:
I think that's a good idea. 

Since appium tests were not a requirement added to the original ticket, none of the other sample apps we have been building have that requirement added. When we decide to work on it, we can discuss implementation ideas at that time for how we accomplish this.